action.js

```diff
@@ -92,8 +92,10 @@ const action = async () => {
 
         const existingChecks = await octokit.checks.listForRef(listExistingChecks);
 
-        if (!(existingChecks.data && existingChecks.data.check_runs.length == 1)) {
```

Comment:
Proper commit message would be: "Do not fail when multiple checks with given name exist" (like PR name)